torch/_dynamo/output_graph.py

```diff
@@ -1617,6 +1628,38 @@ def update_used_symbols(used_symbols, fake: Union[torch.SymInt, torch.Tensor]):
                     # Make sure we delete later occurrences of the same symbol
                     used_symbols.remove(symbol)
 
+    def remove_tensorify_specialized_graphargs(self) -> None:
+        # This is a pretty interesting function. Basically we have this problem
+        # where our compiler tends to choke when we have unused inputs. The way
+        # we support dynamic float arguments is by doing a joint fx pass and
+        # tensorifying away as many symfloats as we can. For the remaining symfloats
+        # we have no choice but to specialize... HOWEVER at that point in time
+        # we can no longer remove graph inputs. So our sledgehammer solution is to
+        # save the state of what inputs we should have specialized in dynamo and
+        # restart analysis. This function incorporates this "view from the future"
+        # state and specializes inputs that we know we won't be able to tensorify
+        # away in the joint pass.
```

Comment:
So explicitly say that in principle we shouldn't choke and so this is not necessary